user-manager-service/src/main/java/com/crmfoodestablishment/usermanager/crud/repository/UserRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.crmfoodestablishment.usermanager.crud.repository;
```

Comment:
pls edit package name
use dot between words + imo 'crud' is not a good name for the package

Reply:
resolved